### ISSUE TITLE
AwakenedTrade Ctrl+D triggers Trades Companion InstantWhisper (?)

### ISSUE THREAD

**Issue by sclient** [lemasato/POE-Trades-Companion#279]
I had this happen once with another item (didnt make a note of it), but I turned off the catch for ctrl-d with trade macro, since I am using awakened trade now instead, and for some reason, for certain items, Companion seems to interpret that keypress as if I had ctrl clicked a whisper.

Doesnt even happen with all the div cards (and I assume there has to be a previous whisper in its buffer, because it gives the last 5 whispers error), but I couldnt get the Humility card to price search by any means.

**Comment by lemasato** [comment]
Hi!
Sorry for the (very) late reply.  

I believe this "AwakenedTrade" restores the clipboard content after searching the item. And since you're pressing CTRL, Trades Companion thinks you just copied this whisper and it triggers the Instant Whisper feature. Though, Instant Whisper already contains something to prevent this from happening. The last 5 whispers are remembered and if you try to send them again, you will get a notification that it was canceled instead as it was sent recently.

So basically, if everything is working as it should, the whisper should NOT be sent after doing Ctrl+D with "AwakenedTrade". Instead, you should get a small tooltip saying that it was canceled. If this is exactly what's happening and the tooltip is annoying to you, then there isn't much more that I can do on my side. It would be best for you to ask them to include an option to prevent the clipboard content to be restored.

**Comment by derschmaal** [comment]
I had the same issue but its not from Awakened Trade. Its from Trade Companion . There is an  Option to toggle on/off instant whispers with strg. By default its on . So on every program what put stuff to your cache , Trade Companion will see it as trade Whisper and post it . Same thing was with a live search tool . Sry for my bad english xD

**Comment by SnosMe** [comment]
@lemasato Hi, I strongly believe that Ctrl functionality should be opt-in instead of opt-out. It would also be nice to add tooltip that this functionality can conflict with other 3-rd party tools.
